examples/scripts/generate-standalone-files.mjs

```diff
@@ -255,11 +255,32 @@ ${exampleClass.example.toString()}
             data = new observer.Observer({});
             main(files);
         }
+        /**
+         * Native error handling triggered by an error. We send them to UI so we can
+         * display them in Monaco editor or elsewhere.
+         */
+        function nativeError(event) {
+            const {lineno, colno, filename, error, message} = event;
```

Comment:
Curly braces should have spaces after the leading `{` and before the trailing `}` if containing more than one element.